desktop/apps/personalize/components/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { ContextProvider } from '@artsy/reaction-force/dist/Components/Artsy'
+import Wizard from '@artsy/reaction-force/dist/Components/Onboarding/Wizard'
+import CollectorIntent from '@artsy/reaction-force/dist/Components/Onboarding/Steps/CollectorIntent'
+import Artists from '@artsy/reaction-force/dist/Components/Onboarding/Steps/Artists'
+import Genes from '@artsy/reaction-force/dist/Components/Onboarding/Steps/Genes'
+import Budget from '@artsy/reaction-force/dist/Components/Onboarding/Steps/Budget'
+const sd = require('sharify').data
+
+export default class App extends React.Component {
```

Comment:
We should try to no longer use default exports; we get way better interoperability and intellisense by using named exports only (`export class App`)